bedrock/firefox/templates/firefox/products/developer.html

```diff
@@ -0,0 +1,269 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/shared" %}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Firefox Developer Edition')}}{% endblock %}
+{% block page_favicon %}{{ static('img/firefox/developer/favicon.png') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/firefox/developer/favicon-196.png') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/developer/page-image.png') }}{% endblock %}
+
+{% block page_desc %}
+  {{_('Built for those who build the Web. The only browser made for developers like you.')}}
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_developer_2017' %}
+{% endblock %}
+
+{% block body_id %}firefox-developer{% endblock %}
+{% block body_class %}firefox-developer{% endblock %}
+
+{% block body_attrs %}{{ super() }} data-global-nav-current-link="firefox"{% endblock %}
```

Comment:
Doh. Removing the global nav was a last-second change, I need to clean up after myself.